public/app/features/variables/query/QueryVariableEditor.tsx

```diff
@@ -237,7 +238,11 @@ export class QueryVariableEditorUnConnected extends PureComponent<Props, State>
           </div>
         </div>
 
-        <SelectionOptionsEditor variable={this.props.variable} onPropChange={this.onSelectionOptionsChange} />
+        <SelectionOptionsEditor
+          variable={this.props.variable}
+          onPropChange={this.onSelectionOptionsChange}
+          onMultiChanged={this.props.changeMulti}
+        />
```

Comment:
not sure why all this new logic was needed, onSelectionOptionsChanged - > 
  this.props.onPropChange({ propName, propValue, updateOptions: true }); 

Sets updateOptions true, can't the state handler for above frunction check multi = true and update state accordingly?

Reply:
> Looks ok, but not sure why this fix is needed in so many of the variable types, looks like changeMultiTo is called from both the shared reducer and each type reducer

Valid point. Hmm, see if I can move the `changeMultiTo` from the shared reducer. Doesn't feel right to have that if-statement in that shared reducer.

Reply:
This is common option better to have it there  it duplicated in 3 variable types